tests/localosmosis/scripts/setup.sh

```diff
@@ -180,60 +206,29 @@ then
     enable_cors
 fi
 
-if [[ $STATE == 'true' ]]
-then
-    # Enter the script folder
-    cd cl-genesis-positions
-
-    # Build script with dependencies
-    apk add --no-cache \
-    ca-certificates \
-    build-base \
-    linux-headers
-    go mod download
-    WASMVM_VERSION=$(go list -m github.com/CosmWasm/wasmvm | cut -d ' ' -f 2) && \
-    wget https://github.com/CosmWasm/wasmvm/releases/download/$WASMVM_VERSION/libwasmvm_muslc.$(uname -m).a \
-        -O /lib/libwasmvm_muslc.a && \
-    # verify checksum
-    wget https://github.com/CosmWasm/wasmvm/releases/download/$WASMVM_VERSION/checksums.txt -O /tmp/checksums.txt && \
-    sha256sum /lib/libwasmvm_muslc.a | grep $(cat /tmp/checksums.txt | grep $(uname -m) | cut -d ' ' -f 1)
-    go mod tidy
-    go build \
-        -mod=readonly \
-        -tags "netgo,ledger,muslc" \
-        -ldflags \
-            "-X github.com/cosmos/cosmos-sdk/version.Name="osmosis" \
-            -X github.com/cosmos/cosmos-sdk/version.AppName="osmosisd" \
-            -X github.com/cosmos/cosmos-sdk/version.Version=${GIT_VERSION} \
-            -X github.com/cosmos/cosmos-sdk/version.Commit=${GIT_COMMIT} \
-            -X github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger,muslc \
-            -w -s -linkmode=external -extldflags '-Wl,-z,muldefs -static'" \
-        -trimpath \
-        -o script \
-        .
-
-    # Get relevant data is not present on the mounted volume.
-    if [[ ! -f "genesis.json" ]]; then
-        if [[ ! -f "subgraph_positions.json" ]]; then
-            echo "Getting concentrated liquidity data from Uniswap subgraph"
-            ./script --operation 0 --localosmosis
-        fi
-
-        echo "Generating Osmosis genesis for the concentrated liquidity module from Uniswap data"
-        ./script --operation 1 --localosmosis
-    fi
-
-    # Run genesis merge script
-    ./script --operation 2 --localosmosis
-    cd ..
-fi
-
 osmosisd start --home $OSMOSIS_HOME &
 
 if [[ $STATE == 'true' ]]
 then
-    create_two_asset_pool "nativeDenomPoolA.json"
-    create_two_asset_pool "nativeDenomPoolB.json"
+    echo "Creating pools"
+    
+    echo "uosmo / uusdc balancer"
+    create_two_asset_pool "uosmoUusdcBalancerPool.json"
+    
+    echo "uosmo / uion balancer"
+    create_two_asset_pool "uosmoUionBalancerPool.json"
+    
+    echo "uweth / uusdc stableswap"
+    create_stable_pool
+    
+    echo "uusdc / uion balancer"
+    create_two_asset_pool "uusdcUionBalancerPool.json"
+
+    echo "stake / uion / uosmo balancer"
     create_three_asset_pool
+
+    echo "uion / uosmo concentrated"
+    create_concentrated_pool
+    create_concentrated_pool_positions
```

Comment:
Note: all of these are broken after the unfork. However, all changes are tested on the v20.x release line.

Made issue to track this: https://github.com/osmosis-labs/osmosis/issues/6979